Dockerfile

```diff
@@ -1,33 +1,35 @@
-FROM golang:1.17-alpine
+FROM golang:1.17-alpine as base
 MAINTAINER team@pganalyze.com
 
-RUN adduser -D pganalyze pganalyze
 ENV GOPATH /go
 ENV HOME_DIR /home/pganalyze
 ENV CODE_DIR $GOPATH/src/github.com/pganalyze/collector
 
-COPY . $CODE_DIR
-WORKDIR $CODE_DIR
-
-# We run this all in one layer to reduce the resulting image size
-RUN apk add --no-cache --virtual .build-deps make curl libc-dev gcc go git tar \
+RUN apk add --no-cache --virtual .build-deps make curl libc-dev gcc git tar \
   && apk add --no-cache ca-certificates setpriv \
```

Comment:
I'm not sure if the ca-certificates package is required on the runtime, if so please let me know and I'll add it to the slim stage.

Reply:
Good point, yes, the Go runtime can use these to validate HTTP certificates (e.g. when calling the API), so these do need to be present at runtime.